README.md

```diff
@@ -9,6 +9,10 @@
 
 Head over to the [Twilio CLI documentation](https://www.twilio.com/docs/twilio-cli/quickstart).
 
+## Architecture and Functional Overview
+
+Head over to the [CLI Architecture and Functional Overview](https://github.com/twilio/twilio-cli/blob/CLI_doc/ARCHITECTURE.md)
```

Comment:
this link needs to be updated, from `CLI_doc` to `main` for main branch

Reply:
FIxed this, thanks!